src/tree_sitter/alloc.h

```diff
@@ -9,20 +9,7 @@ extern "C" {
 #include <stdbool.h>
 #include <stdio.h>
 
-#include "nvim/memory.h"
-
-#if 1
-
-static inline bool ts_toggle_allocation_recording(bool value) {
-  return false;
-}
-
-#define ts_malloc xmalloc
-#define ts_calloc xcalloc
-#define ts_realloc xrealloc
-#define ts_free xfree
-
-#elif defined(TREE_SITTER_TEST)
```

Comment:
Is this section no longer needed? It is confusing to me why it was there but no longer is.

Reply:
Actually, I keeped it last time because I thought it was mandatory, but visibly it is not anymore...
I can add it if you want though but I'm not sure of its usefullness...

Reply:
Are any of those `#DEFINE`s used anywhere? Just like grepping through the source or anything like that.

Reply:
cc @bfredl if you're around.